.github/workflows/android.yml

```diff
@@ -1,9 +1,11 @@
 on:
   push:
+  pull_request:
```

Comment:
For the Kotlin SDK we're doing [this](https://github.com/powersync-ja/powersync-kotlin/blob/1d939cc76e9501a5d81dfb8d7c843ba2f66e46ac/.github/workflows/test.yml#L3-L8) which may be nice to avoid the "skipped" list of actions on every PR we open now:

```YAML
on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]
  workflow_call:
```

It would have the downside that our branches only get tested after we open a PR, but that migh be fine.

Reply:
I'm annoyed that github doesn't have something better built-in for this. I also used the workaround here in [powersync-service](https://github.com/powersync-ja/powersync-service/pull/205), and it seems to work quite well: Same behavior as currently (build on each push) for branches within the repo, and switches to the "pull_request" builds when it's from a fork. Main downside is the extra boilerplate in the workflows.